x-pack/plugins/infra/public/alerting/inventory/rule_data_formatters.ts

```diff
@@ -5,20 +5,12 @@
  * 2.0.
  */
 
-import { i18n } from '@kbn/i18n';
-import { ALERT_ID } from '@kbn/rule-data-utils';
+import { ALERT_REASON } from '@kbn/rule-data-utils';
 import { ObservabilityRuleTypeFormatter } from '../../../../observability/public';
 
 export const formatReason: ObservabilityRuleTypeFormatter = ({ fields }) => {
-  const groupName = fields[ALERT_ID];
-  const reason = i18n.translate('xpack.infra.metrics.alerting.inventory.alertReasonDescription', {
-    defaultMessage: 'Inventory alert for {groupName}.', // TEMP reason message, will be deleted once we index the reason field
-    values: {
-      groupName,
-    },
-  });
-
-  const link = '/app/metrics/inventory';
+  const reason = fields[ALERT_REASON] ?? '';
+  const link = '/app/metrics/inventory'; // FIXME? Not sure if we need to do something with this link
```

Comment:
@afgomez At the moment we link to the default inventory page. I have created an issue for metrics team to give us the option to directly link to the specific inventory item with the flyout open https://github.com/elastic/kibana/issues/106497.

@weltenwort I see I didn't create a ticket for indexing the node information (if it is a container, pod or host), since link changes accordingly. I'll create one